### PR TITLE
PAAS-2310: add whitelist for SAM module

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-2310

### CODE REVIEW

**Review by kevinvandel — CHANGES_REQUESTED** [review]
While we're at it, since you implemented an easy way to update the config, could you please add database-connector and elasticsearch-connector when adding AS, and jexperience-dashboards and kibana-dashboard-provider when adding jExperience?

**Comment by adoux** [comment]
> While we're at it, since you implemented an easy way to update the config, could you please add database-connector and elasticsearch-connector when adding AS, and jexperience-dashboards and kibana-dashboard-provider when adding jExperience?

I tested and having the modules in the list is not an issue when they are not installed so I got rid of that extra action and added the whole list directly to the SAM config file

File: packages/jahia/migrations/migrate-to-v23.yaml [hunk 1]
```diff
@@ -133,3 +134,23 @@ actions:
...
+        if ! grep -q "probes.ModuleState.whitelist" $sam_conf_file; then
+          modules=(
+            distributed-sessions
+            personal-api-tokens
+            server-availability-manager
+            graphql-dxm-provider
+            org.jahia.bundles.clustering
+            jexperience
+            jexperience-dashboards
+            kibana-dashboards
```
Comment (kevinvandel) [inline packages/jahia/migrations/migrate-to-v23.yaml:150]: Isn't it `kibana-dashboards-provider` instead?

Comment (adoux) [reply packages/jahia/migrations/migrate-to-v23.yaml]: Yes it is, good call! Fixed

**Review by kevinvandel — APPROVED** [review]

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**